the plist_format_t enum is now defined in libplist, and it's typedef'ed

diff -u src-orig/utils.c src/utils.c
--- a/src/utils.c
+++ b/src/utils.c
@@ -302,7 +302,7 @@
 	return 1;
 }
 
-int plist_write_to_filename(plist_t plist, const char *filename, enum plist_format_t format)
+int plist_write_to_filename(plist_t plist, const char *filename, plist_format_t format)
 {
 	char *buffer = NULL;
 	uint32_t length;
diff -u src-orig/utils.h src/utils.h
--- a/src/utils.h
+++ b/src/utils.h
@@ -78,13 +78,8 @@
 int buffer_read_from_filename(const char *filename, char **buffer, uint64_t *length);
 int buffer_write_to_filename(const char *filename, const char *buffer, uint64_t length);
 
-enum plist_format_t {
-	PLIST_FORMAT_XML,
-	PLIST_FORMAT_BINARY
-};
-
 int plist_read_from_filename(plist_t *plist, const char *filename);
-int plist_write_to_filename(plist_t plist, const char *filename, enum plist_format_t format);
+int plist_write_to_filename(plist_t plist, const char *filename, plist_format_t format);
 
 uint64_t mstime64(void);
 void get_tick_count(struct timeval * tv);
